db/migrate/20220714070722_create_reservations.rb

```diff
@@ -0,0 +1,11 @@
+class CreateReservations < ActiveRecord::Migration[7.0]
+  def change
+    create_table :reservations do |t|
+      t.datetime :reservation_time
+      t.references :user, null: false, foreign_key: { to_table: :users }, index:true
+      t.references :motorcycle, null: false, foreign_key: { to_table: :motorcycles }, index:true
+
```

Comment:
- [ ] Please add the Phone number of the user for contact. The phone column is string